scripts/remove

```diff
@@ -34,52 +34,125 @@ app=$YNH_APP_INSTANCE_NAME
 
 domain=$(ynh_app_setting_get $app domain)
 firmware_nonfree=$(ynh_app_setting_get $app firmware_nonfree)
+service_name=$(ynh_app_setting_get $app service_name)
+final_path=$(ynh_app_setting_get $app final_path)
+
+#=================================================
+# STANDARD REMOVE
+#=================================================
+# REMOVE SERVICE FROM ADMIN PANEL
+#=================================================
+
+# Remove a service from the admin panel, added by `yunohost service add`
+if yunohost service status $service_name >/dev/null 2>&1
+then
+	ynh_print_info "Removing $app service"
+	yunohost service stop $service_name
+	yunohost service remove $service_name
+fi
+
+#=================================================
+# STOP AND REMOVE SERVICE
+#=================================================
+ynh_print_info "Stopping and removing the systemd service"
+
+# Remove the dedicated systemd config
+ynh_remove_systemd_config
+
+#=================================================
+# REMOVE DEPENDENCIES
+#=================================================
+ynh_print_info "Removing dependencies"
+
+# Remove metapackage and its dependencies
+ynh_remove_app_dependencies
+
+#=================================================
+# REMOVE APP MAIN DIR
+#=================================================
+ynh_print_info "Removing app main directory"
+
+# Remove the app directory securely
+ynh_secure_remove "$final_path"
+
+ynh_secure_remove /usr/local/bin/$service_name
+
+ynh_secure_remove "/usr/local/bin/iw_multissid"
+ynh_secure_remove "/usr/local/bin/iw_devices"
+ynh_secure_remove "/usr/local/bin/iw_ssids"
+ynh_secure_remove "/usr/local/bin/ipv6_expanded"
+ynh_secure_remove "/usr/local/bin/ipv6_compressed"
+
+ynh_secure_remove "/etc/sudoers.d/${app}_ynh"
 
-# The End
-yunohost service stop ynh-hotspot
-yunohost service disable ynh-hotspot
-yunohost service remove ynh-hotspot
-ynh_secure_remove /etc/systemd/system/ynh-hotspot.service
-ynh_secure_remove /usr/local/bin/ynh-hotspot
 for FILE in $(ls /tmp/.ynh-hotspot-* 2>/dev/null)
 do
     ynh_secure_remove "$FILE"
 done
 
-# Update firewall for DHCP
-yunohost firewall disallow --ipv6 UDP 547
-yunohost firewall disallow UDP 67
-
 # Remove confs
-ynh_secure_remove /etc/dnsmasq.dhcpd
-ynh_secure_remove /etc/nginx/conf.d/${domain}.d/wifiadmin.conf
-ynh_secure_remove /etc/php5/fpm/pool.d/wifiadmin.conf
+ynh_secure_remove "/etc/dnsmasq.dhcpd/dhcpdv6.conf.tpl"
+ynh_secure_remove "/etc/dnsmasq.dhcpd/dhcpdv4.conf.tpl"
 for FILE in $(ls /etc/hostapd/hostapd.conf{.tpl?,} 2>/dev/null)
 do
     ynh_secure_remove "$FILE"
 done
 ynh_secure_remove /etc/sudoers.d/hotspot_ynh
```

Comment:
Now it's already done a few lines up so that one's a duplicate, creating this warning : 

```
Warning: /etc/sudoers.d/hotspot_ynh wasn't deleted because it doesn't exist.
```

Reply:
Thank you! :-)